aodndata/moorings/handlers.py

```diff
@@ -93,29 +93,41 @@ def process(self):
         """Handle a zip file containing images and no NetCDF files. In this case we just want to publish the zip file
         itself, not the individual images. If we encounter a "mixed" zip file with images and netCDF files,
         we're just going to give up, for now.
-        """
+        """        
         images = PipelineFileCollection(f for f in self.file_collection if f.file_type.is_image_type)
         netcdfs = self.file_collection.filter_by_attribute_id('file_type', FileType.NETCDF)
         is_zip = self.file_type is FileType.ZIP
         have_images = len(images) > 0
         have_netcdfs = len(netcdfs) > 0
-        if is_zip and have_images:
-            if have_netcdfs:
-                raise InvalidFileContentError(
-                    "Zip file contains both images and netCDFs. Don't know what to do!"
-                    " They are handled differently, so please upload only one at a time."
-                )
-            if not DwmFileClassifier.SOTS_IMAGES_ZIP_PATTERN.match(self.file_basename):
-                raise InvalidFileNameError(
-                    "Zip file contains images, but its name does not match pattern for images zip file "
-                    "(regular expression '{p}')".format(p=DwmFileClassifier.SOTS_IMAGES_ZIP_PATTERN.pattern)
+        is_image_zip_pattern = is_zip and DwmFileClassifier.SOTS_IMAGES_ZIP_PATTERN.match(self.file_basename)
+        is_calibration_zip_pattern = is_zip and DwmFileClassifier.SOTS_CALIBRATION_ZIP_PATTERN.match(self.file_basename)
+        zip_file_msg = 'images' if is_image_zip_pattern else 'calibration'
+             
+        if have_netcdfs:
```

Comment:
This will raise an error for every valid upload of a zip file containing _only_ netCDFs!